shared/index.native.js

```diff
@@ -89,6 +89,13 @@ class Keybase extends Component {
 }
 
 function load () {
+  // Native String.startswith() sometimes incorrectly returns false on Android!
+  // $FlowIssue redefining startsWith
+  String.prototype.startsWith = function (searchString, position) { // eslint-disable-line no-extend-native
```

Comment:
likely better to do it in globals.native. we import this on top of this file where we do some polyfilling already